### PR TITLE
Add DR test profile on Nutanix for 4.16

### PR DESCRIPTION
**Pull request by wangke19**
Add DR test profile on Nutanix for OCP 4.16. Associated Jira issue https://issues.redhat.com/browse/OCPQE-20106

### CODE REVIEW

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-ovn-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-ovn-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-ovn-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-f14-disasterrecovery

**Comment by wangke19** [comment]
Looks like also vpn issue, we are still waiting DPP to resove it, detail see: https://redhat-internal.slack.com/archives/CF8SMALS1/p1715153995732219?thread_ts=1715062181.493839&cid=CF8SMALS1

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-ovn-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-ovn-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-ovn-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-ovn-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse  periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-nutanix-ipi-proxy-f14-disasterrecovery

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 1]
```diff
@@ -2200,6 +2200,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-ui
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy
+- as: nutanix-ipi-proxy-f14-disasterrecovery
+  cluster: build01
+  cron: 12 20 3,17 * *
+  steps:
+    cluster_profile: nutanix-qe-dis
+    env:
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:2208]:
```suggestion
    env:
        EXTRACT_MANIFEST_INCLUDED: "true"
```

Comment (wangke19) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:2208]: Updated.

**Pull request merged**